runtime/kusama/src/lib.rs

```diff
@@ -99,6 +99,13 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	transaction_version: 4,
 };
 
+/// The BABE epoch configuration at genesis.
+pub const BABE_GENESIS_EPOCH_CONFIG: babe_primitives::BabeEpochConfiguration =
+	babe_primitives::BabeEpochConfiguration {
+		c: PRIMARY_PROBABILITY,
+		allowed_slots: babe_primitives::AllowedSlots::PrimaryAndSecondaryVRFSlots
```

Comment:
Currently I think Kusama is still using `PrimaryAndSecondaryPlainSlots`, I think if we set it `PrimaryAndSecondaryVRFSlots` here we will ruin the migration below. So let's keep it as `PrimaryAndSecondaryPlainSlots` until we actually migrate the chain to use secondary VRF slots.

Reply:
Well it's `PrimaryAndSecondaryVRFSlots` in the `BabeGenesisConfiguration`. I'm not sure if it's different on-chain though.

Reply:
This was changed to `PrimaryAndSecondaryVRFSlots` recently in #2480, but on-chain it's still `PrimaryAndSecondaryPlainSlots`. We should really revert to `PrimaryAndSecondaryPlainSlots` until we change it on-chain, and in this PR in particular because otherwise `EpochConfig` will not reflect the actual value that the client is expecting.

Reply:
Okay. I'll change it in the migration function but not in the genesis config.